botorch/models/kernels/contextual_sac.py

```diff
@@ -52,6 +46,15 @@ def __init__(
         batch_shape: torch.Size,
         device: Optional[torch.device] = None,
     ) -> None:
+        """
+        Args:
+            decomposition: Keys are context names. Values are the indexes of parameters
+                belong to the context. The parameter indexes are in the same order
+                across contexts.
+            batch_shape: Batch shape as usual for gpytorch kernels.
+            device: The torch device. Defaults to None.
```

Comment:
I think in general we don't need to provide default values that are clear from the type annotations. It's a different story when e.g. some default not clear form the annotations is chosen if a `None` value is provided. 
```suggestion
            device: The torch device.
```

Reply:
Yep makes sense. A robot wrote these.